conf/messages.en

```diff
@@ -2228,7 +2228,7 @@ whichYearIsLossReported.change.hidden = Change in which tax year you most recent
 
 ########## National Insurance Contributions ##########
 
-nics.exemptForDifferentReason = I’m exempt for a different reason
+nics.exemptForDifferentReason = None of these businesses are exempt
 nics.diverDivingInstructor.individual = You are a diver or diving supervisor
```

Comment:
Need both messages, can do 
```
nics.class-four-other-exemption = I’m exempt for a different reason
nics.class-four-none-exempt = None of these businesses are exempt
```
Then in the `viewmodels/checkanswers/nics/package` object can change the if-else to :
```
val values =
          if (idList.contains(classFourOtherExemption)) Seq(messages(s"nics.$classFourOtherExemption"))
      else if (idList.contains(classFourNoneExempt)) Seq(messages(s"nics.$classFourNoneExempt"))
      else getTradingNamesFromIds(idList, businesses)
    values.mkString(",<br>")
    ```

Reply:
Then in app/views/journeys/nics/Class4DivingExemptView.scala.htm:
```exclusiveCheckboxText = Some(s"nics.$classFourOtherExemption"),
            exclusiveCheckboxValue = Some(classFourOtherExemption.value)```
            
and in app/views/journeys/nics/Class4NonDivingExemptView.scala.html:
```exclusiveCheckboxText = Some(s"nics.$classFourNoneExempt"),
            exclusiveCheckboxValue = Some(classFourNoneExempt.value)```